tor_archivist/main.py

```diff
@@ -98,9 +100,15 @@ def run(cfg):
             continue
 
         # the original post that might have been transcribed
-        original_post = config.r.submission(url=post.url)
+        try:
```

Comment:
If a submission is too old but is from a private sub, it looks like we'll skip it. Can we move the check for an old (untranscribed) post to above this so it'll still remove the old shit?

Reply:
Well, the problem is it's throwing an error here on trying to retrieve the submission. We could `None`-out the `original_post` var and carefully make it work, but part of the deal of the archivist is to post to the archive sub with a direct link to the transcription comment. Which we can't do because sub is private.

Reply:
So if we remove the post without cross-posting to the archive, that's an option I suppose. Would that be more desireable?

Reply:
Let's do it this way:

* set it up to remove without crossposting
* add a TODO that says we want to implement the author check from ToR
* use the author check to grab the permalink to the transcription we can't access from the author's history
* use that permalink to generate the crosspost to the archive

Obviously today we won't worry about most of that, but the TODO should at least note that it's a plan?